lib/api.js

```diff
@@ -104,22 +104,15 @@ class API {
    *
    * @async
    * @param {String} docJSONPath - the path to the analytic JSON
-   * @param {Boolean} supportsCPU - whether the analytic supports CPU execution
-   * @param {Boolean} supportsGPU - whether the analytic supports GPU execution
    * @returns {object} an object containing metadata about the posted analytic
    * @throws {APIError} if the request was unsuccessful
    */
-  async uploadAnalytic(docJSONPath, supportsCPU, supportsGPU) {
+  async uploadAnalytic(docJSONPath) {
     let uri = this.baseURL + '/analytics';
     let formData = {
       file: fs.createReadStream(docJSONPath),
     };
-    let params = {
-      supports_cpu: supportsCPU,
-      supports_gpu: supportsGPU,
-    };
-    let body = await requests.post(
-      uri, this.header_, {formData: formData, qs: params});
```

Comment:
As a side note these were not query parameters (e.q. `?supports_cpu=true`); they were form fields as well.